grisera_api/participant_state/participant_state_service_graphdb.py

```diff
@@ -20,9 +19,12 @@ class ParticipantStateServiceGraphDB(ParticipantStateService):
         personality_service (PersonalityService): Service to manage personality models
     """
     graph_api_service = GraphApiService()
-    participant_service = ParticipantServiceGraphDB()
-    appearance_service = AppearanceServiceGraphDB()
-    personality_service = PersonalityServiceGraphDB()
+
+    def __init__(self):
+        self.participant_service = None
+        self.appearance_service = None
+        self.personality_service = None
+        self.participation_service = None
```

Comment:
Missing type definition of these services

Reply:
As above